internal/clientconn/conn.go

```diff
@@ -127,15 +127,15 @@ func newConn(opts *newConnOpts) (*conn, error) {
 //
 // The caller is responsible for closing the underlying net.Conn.
 func (c *conn) run(ctx context.Context) (err error) {
-	var connInfo conninfo.ConnInfo
+	connInfo := new(conninfo.ConnInfo)
```

Comment:
this change doesn't seem to be needed anymore.

Reply:
Fixed.